docusaurus/docusaurus.config.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+    title: 'next-with-moxy',
+    tagline: 'MOXY\'s boilerplate to create Next.js based applications',
+    url: 'https://next-with.moxy.tech',
```

Comment:
This was discussed with @satazor and the idea came up looking to our current domains. Since we have `moxy.tech`, we can use it for tech stuff like this.

However, it's still open for debate.

Reply:
Ah, I agree with the idea then.